apps/notifications/src/panel/templates/note-list.jsx

```diff
@@ -347,28 +347,34 @@ export class NoteList extends Component {
 		};
 
 		return (
-			<div className={ classes } id="wpnc__note-list">
-				<FilterBar controller={ this.props.filterController } />
-				<button className="screen-reader-text" onClick={ this.props.closePanel }>
-					{ this.props.translate( 'Close notifications' ) }
-				</button>
-				<div ref={ this.storeScrollableContainer } className={ listViewClasses }>
-					<ol ref={ this.storeNoteList } className="wpnc__notes" { ...notificationsListAriaProps }>
-						<StatusBar
-							statusClasses={ this.state.statusClasses }
-							statusMessage={ this.state.statusMessage }
-							statusTimeout={ this.state.statusTimeout }
-							statusReset={ this.resetStatusBar }
-						/>
-						{ notes }
-						{ this.props.isLoading && (
-							<div style={ loadingIndicatorVisibility } className="wpnc__loading-indicator">
-								<Spinner />
-							</div>
-						) }
-					</ol>
+			<>
+				<StatusBar
```

Comment:
Move to outside to ensure the message is visible when the user is reading a single notification message